backend/chunker/chunker.go

```diff
@@ -218,6 +218,23 @@ It has the following fields: ver, size, nchunks, md5, sha1.`,
 					Help:  "Warn user, skip incomplete file and proceed.",
 				},
 			},
+		}, {
+			Name:     "transactions",
+			Advanced: true,
+			Default:  "rename",
+			Help:     `Choose how chunker should handle temporary files during transactions.`,
+			Examples: []fs.OptionExample{
+				{
+					Value: "rename",
+					Help:  "Rename temporary files after a successful transaction.",
```

Comment:
Tracking transaction-marked chunks is obviously impossible without metadata.
Let's give user a hint.
Also, we are not going to implement `norename` for for Copy/Move in this PR and have unresolved compatibility problems with previous versions of rclone. I suggest to add a warning like `this method is EXPERIMENTAL, don't use on production systems`.

Reply:
It's better than nothing.
Though manytimes users will ignore warnings and loose data. Enjoy https://github.com/rclone/rclone/issues/1574#issuecomment-624623587 and #4298.

Reply:
Please hide this option from command line:
[visibility value](https://github.com/rclone/rclone/blob/v1.53.3/fs/fs.go#L131)
[how to use visibility values](https://github.com/rclone/rclone/blob/v1.53.3/backend/googlecloudstorage/googlecloudstorage.go#L98)
(I should have done this to the _metadata_format_)